services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker_process_messages.py

```diff
@@ -60,7 +61,7 @@ async def _process_start_event(
 ):
     service_type = (
         ResourceTrackerServiceType.COMPUTATIONAL_SERVICE
-        if msg.service_type == "COMPUTATIONAL"
+        if msg.service_type == ServiceType.COMPUTATIONAL
```

Comment:
SEE the differences bewteen `enum.Enum` and  inheriting from (`str`,`enum.Enum`) in 

https://github.com/pcrespov/sandbox-python/blob/python-stuff/test_enumerations.py